Clean up AutoOps notification settings page (#6944)
elastic/docs-content: deploy-manage/monitor/autoops/ec-autoops-notifications-settings.md

2. Create a new service (or open an existing one) for AutoOps alerts.
3. Assign an escalation policy to specify who gets paged.
4. On the service **Integrations** tab, click **Add another integration**.

> 5. select [Events API v2](https://developer.pagerduty.com/docs/events-api-v2-overview).

Reviewer: ```suggestion
5. Select [Events API v2](https://developer.pagerduty.com/docs/events-api-v2-overview).
```

6. Expand the integration and copy the integration key (also known as the routing key for Events API v2).
7. Store the key securely. You will need it when configuring the connector in AutoOps.

**AutoOps configuration**
1. Follow the instructions to [set up a connector](#ec-setup-autoops-connectors) and select **PagerDuty** as your connector type.
2. In the **Key** field, paste the integration key you copied during the PagerDuty configuration.
3. Add a [notification filter](#ec-add-notification-filters) for this connector.
:::

:::{dropdown} Slack
:name: ec-autoops-slack

The Slack integration consists of the following parts:

**Slack configuration**

1. Go to [Slack API](https://api.slack.com/apps)
2. Click **Create an App** > **From scratch**.
3. Enter an app name and select your workspace, and then click **Create App**.
4. From the left menu, select **Incoming Webhooks**.
5. Turn on **Activate Incoming Webhooks**.
6. Click **Request to Add New Webhook** and submit a request to your organization's admins. 
7. When your request is approved, select a Slack channel from the list of **Webhook URLs for Your Workspace** and click **Allow**.
8. Copy the webhook URL.

> 9. Store the URL securely. You will need it when configuring the connector in AutoOps.

Reviewer: there are lots of steps here that could age. is there a way to link off to official docs for the other product and then just indicate the key settings? this applies to all external procedures on this page.

Author: I considered this, but unfortunately not all the external products have clear documentation about this. I've replaced some sections with external links, but the rest of them will have to stay

**AutoOps configuration**
1. Follow the instructions to [set up a connector](#ec-setup-autoops-connectors) and select **Slack** as your connector type.
2. In the **URL** field, paste the URL you copied during the Slack configuration.
3. Add a [notification filter](#ec-add-notification-filters) for this connector.

:::

> :::{dropdown} VictorOps

Reviewer: should this dropdown also have splunk on-call in it

Author: I opted not to add it because the UI just says "VictorOps"

2. In the **URL** field, paste the URL you copied during the Splunk On-Call configuration.
3. Add a [notification filter](#ec-add-notification-filters) for this connector.

:::

:::{dropdown} Opsgenie
:name: ec-autoops-opsgenie

The Opsgenie integration consists of the following parts:

**Opsgenie configuration**

1. Open the main page of your Opsgenie account and click the **Teams** tab. Make sure a team is defined.
2. Go to **Settings** > **Integrations**.
3. Click **Add Integration**. In the **Integration List**, search for **API**.
4. Complete the fields and click **Save**.
5. Copy the API key and store it securely. You will need it when configuring the connector in AutoOps.

**AutoOps configuration**

1. Follow the instructions to [set up a connector](#ec-setup-autoops-connectors) and select **Opsgenie** as your connector type.
2. In the **API key** field, paste the API key you copied during the Opsgenie configuration.
3. Add a [notification filter](#ec-add-notification-filters) for this connector.
:::

:::{dropdown} Microsoft Teams
:name: ec-autoops-ms-configuration

> The Microsoft Teams integration consists of the following parts:

Reviewer: this pattern feels a little less than ideal to me because the second part is so far away from the colon. A more valuable intro would be something like "To set up the Microsoft Teams integration, you need to perform configurations in Microsoft Teams and AutoOps." or something

not too picky about this, just figure it's a good time to mention it because you're editing all of these lines

Author: Good point. Updated everywhere

**Microsoft Teams configuration**

> 1. Follow Microsoft Teams' instructions to create an [incoming webhook](https://docs.microsoft.com/en-us/microsoftteams/platform/webhooks-and-connectors/how-to/add-incoming-webhook).

Reviewer: 🔥 more of this

    * **Method**: Select a method. `POST` is selected by default.
    * **Header**: Add a key and value. `Content-Type` and `application/json` are added by default.
3. In the **Body** section, edit the message according to how you want it to appear in notifications. Refer to your application documentation for the expected message schema. AutoOps provides a set of optional fields to use. 

    * `RESOURCE_ID`: Customer deployment ID
    * `RESOURCE_NAME`: Customer deployment name
    * `TITLE`: The title of the event.
    * `DESCRIPTION`: The description of the issue that was found.
    * `SEVERITY`: One of the three severity levels (High, Medium, Low).
    * `STATUS`: Indicates if the event is currently open or closed.
    * `MESSAGE`: The background and impact of the issue.
    * `START_TIME`: The time the event was opened.
    * `END_TIME`: The time the event was closed.
    * `ENDPOINT_TYPE`: The type of endpoint (PagerDuty, Slack, VictorOps, Opsgenie, or Microsoft Teams).
    * `AFFECTED_NODES`: List of affected nodes.
    * `AFFECTED_INDICES`: List of affected indices.
    * `EVENT_LINK`: Direct link to the event in AutoOps.

> 4. Optionally, test the webhook integration by using the [webhook.site](https://webhook.site/#!/view/fe9d630e-2f01-44b7-9e41-ef9520fbe9a7).

Reviewer: are we opinionated about this tool? could we say "a webhook testing tool like webhook.site" ? 

```suggestion
4. Optionally, test the webhook integration by using [webhook.site](https://webhook.site/#!/view/fe9d630e-2f01-44b7-9e41-ef9520fbe9a7).
```

:::

### Add notification filters [ec-add-notification-filters]

After adding a connector, add a notification filter to specify which events to receive notifications for and how you want to be notified. You can create an unlimited number of filters, and the same connector can be used across multiple filters.

To add a filter, follow these steps:

1. On the **Notifications settings** page, navigate to the **Filter settings** tab and click **Add filter**.
2. Enter a name that best describes the type of notification. This name will appear in other reports and dashboards.
3. Select the connectors to receive the notification. 
4. Select the deployments for which this filter should trigger notifications.
5. Use the **Delay** field to set the period of time you want AutoOps to wait before sending the notification. If all the events listed in this filter are closed by AutoOps in this time, no notification will be sent.
6. Use the **Included Events** and **Excluded Events** fields to select which events should trigger or not trigger this notification.
7. Save your settings.

## Notifications report [ec-notification-report]

> From the **Notifications** report, you can check all the notifications sent. The report lists all the events that were set up in the notification filters and their status.

Reviewer: maybe?
```suggestion
From the **Notifications** report, you can review all of the notifications that have been sent by AutoOps. The report lists all the events that were set up in the notification filters and their status.
```
